### PR TITLE
Moved speakers to subpages

### PR DESCRIPTION
**Pull request by sparkica**
Speakers are now added as subpages. This means:
- their details are now available under `/speakers/_speaker_name_`
- editing _speakers.ini_ databag will have **no effect** from now on. If you need to edit speaker's info, edit in in contents/_speaker_name_.

Also, I've added default profile img to template, so there's no need to specify one in the contents.lr if the speaker didn't provide one. Just put the `image:` key there.

About design changes:
It's not perfect, but it does the job.
I had to add additional _block_ to layout, because I had to tweak the header gradient for subpages. Otherwise the subpage is formatted same as it was before. I just moved some classes in `_components.scss` in special section. 

And the last: if someone tries to reach `/speakers`, they're redirected to /#speakers. It made no sense to show the only list of speakers...

### CODE REVIEW

**Comment by sparkica** [comment]
@ercchy review please...

File: templates/speaker.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        <div class="medium-12 column">
+            <div class="row">
+                <div class="medium-10 column">
+                    <img src="{{ this.image|default("/images/speakers/default.png", true) | asseturl }}" class="modal-img-circle">
+                    <span class="speaker-name">{{ this.name }}</span>
+                </div>
+            </div>
+            <div class="row">
+                <div class="medium-12 column">
+                    <h2>Session: {{ this.talk_title }}</h2>
```
Comment (zanderle) [inline templates/speaker.html:18]: Je potreben 'Session' tukej? Mogoce kar samo talk title?

File: templates/speakers.html [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{% set root = site.get_root() %}
+<meta http-equiv="refresh" content="0;url={{ root|url }}#speakers">
```
Comment (zanderle) [inline templates/speakers.html:2]: lektor nima nobene opcije za mal lepsi redirect ane? :/ Mal smotano, ampak ok

**Review by zanderle — COMMENTED** [review]
Tole zgleda super. Smiselno bi bilo potem speakers databag odstranit in tudi na front page-u brati kar iz speaker folderja. Ce ne je samo vprasanje casa kdaj bomo spremenili samo nekje in ne povsod.

File: templates/speaker.html (hunk 1 above)
Comment (sparkica) [reply templates/speaker.html:18]: To je bilo kopirano iz obstoječe vsebine. Vsebine nisem spreminjala.

File: templates/speakers.html (hunk 2 above)
Comment (sparkica) [reply templates/speakers.html:2]: Ne, zaenkrat ne...

**Comment by sparkica** [comment]
@zanderle Na glavni strani se je že bralo iz speakers folderja - nikjer več se ni uporabljal databag. Pustila sem ga, če komu ni bi bila všeč ta sprememba 😎 Zdaj je odstranjen.

**Pull request merged**